src/screens/Customers/AddCustomer.tsx

```diff
@@ -52,16 +77,24 @@ function AddCustomer(props: AddCustomerProps) {
     setSelectedTariffPlanId(event.target.value as string);
   }
 
+  const handleSelectMeterType = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setSelectedMeterType(event.target.value as MeterType);
+  }
+
   // TODO: Add form input validation and error messaging
   const handleSubmit = (event: React.MouseEvent) => {
+    // Set button as loading
+    setLoading(!loading);
```

Comment:
Actually, you can disregard this comment for now (no need to do error handling) since we're talking with NBJC about how possibly using a standardized library for it